### DISCUSSION TITLE
Special folder

### DISCUSSION THREAD

**Discussion by CoAtef** [microsoft/mfcmapi#459] [Q&A]
Hey,
I'm facing a problem with an online inbox that has been moved down under the deleted items folder to be a subfolder to it; I'm not expert with MFCmapi tool but when I'm tried to copy and paste the inbox folder to be a seperated folder out of the deleted items folder I've faced the following error:
Error: CopyFolder failed 
Code: Mapi_E_NO_ACCESS == 0x80070005
function (null)
File D:\a\1\s\UI\Dialogs\HierarchyTable\MsgStoreDlg.cpp
Line 479

**Comment by stephenegriffin** [comment]
You'd probably need to pass the move flag to have a hope of making that work, but if it doesn't that's a server issue. There might be some powershell commands that can help

**Comment by CoAtef** [comment]
Hello,
Actually I’ve already deleted the rules and I’m sure there are no rules there, although the problem still exist; so could you please provide me with the powershell commands that help me in such a matter?
